src/librustc_typeck/diagnostics.rs

```diff
@@ -2804,6 +2814,16 @@ trait Trait {
 }
 
 type Foo = Trait<Bar=i32>; // ok!
+
+// or:
+
+trait Trait {
```

Comment:
You cannot put the same trait name in this same block, otherwise the test will fail. Either you change the trait's name or you create a new code block.